packages/cardano-services-client/test/HandleProvider/handleHttpProvider.test.ts

```diff
@@ -0,0 +1,29 @@
+import { handleHttpProvider } from '../../src';
+import { logger } from '@cardano-sdk/util-dev';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+const config = { baseUrl: 'http://some-hostname:3000/handle', logger };
+
+describe('handleHttpProvider', () => {
+  let axiosMock: MockAdapter;
+
+  beforeAll(() => {
+    axiosMock = new MockAdapter(axios);
+  });
+
+  afterEach(() => {
+    axiosMock.reset();
+  });
+
+  afterAll(() => {
+    axiosMock.restore();
+  });
+
+  test("resolve handles doesn't throw", async () => {
```

Comment:
'Resolve handles doesn't throw an error when handles come empty'

Reply:
fde4064eb should solve this